model/validation.go

```diff
@@ -49,7 +49,8 @@ func IsDNS1123Label(value string) bool {
 	return len(value) <= dns1123LabelMaxLength && dns1123LabelRex.MatchString(value)
 }
 
-func validatePort(port int) error {
+// ValidatePort ...
+func ValidatePort(port int) error {
```

Comment:
Why export all of Validate*() functions? I don't see them used outside of the model package. If this is prep work for a future CL then the function comments should be completed.